snuba/settings/__init__.py

```diff
@@ -18,6 +18,7 @@
 
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
+CLICKHOUSE_MAX_QUERY_SIZE_BYTES = 256 * 1024  # 256 KiB by default
```

Comment:
Putting this in settings means that whoever is running Snuba would be able to configure and override our default value. Personally I don't see a strong use case for supporting that since the main purpose of this is just to avoid the max ClickHouse query size. I think this would make more sense just hardcoded in our codebase somewhere outside of settings. What do you think?

Reply:
I just needed a place to hardcode that value and settings seemed appropriate. Eventually this value would also be used to reject queries that are too long before they're passed to Clickhouse. I can define this constant in `snuba/web/query.py` itself since the settings are meant to be configurable?

Reply:
Putting it in `snuba.web.query` seems good to me